_posts/en/posts/2017-09-01-release-0.15.0.md

```diff
@@ -0,0 +1,130 @@
+
+## Multiwallet
+
+In Bitcoin Core 0.15.0, a single running Bitcoin Core program can now manage multiple wallets with ease.  This feature is still new and only accessible to expert users, but we hope to make it available in the graphical user interface in the future.
+
+You can use the new multiwallet mode to,
+
+- Use one wallet for your business and one wallet for your personal use in order to simplify your accounting and prevent accidental misuse of funds.
+
+- Separate bitcoins that are associated with your identity from bitcoins that can't be traced back to you in order to help protect your privacy.  Each wallet uses completely different private keys and will never automatically mix its bitcoins with bitcoins from another wallet, preventing taint analysis from connecting those two wallets.
+
+- Manage a Bitcoin backend for an organization in much the same way that has been historically possible with the now-deprecated Bitcoin Core accounts features.  As a simple example, if you handle small bitcoin balances for your less-experienced friends and family, you can now manage each person's bitcoins in a separate wallet rather than risking mixing them up with your own bitcoins.
+
+These features are currently only available through the RPC interface for programmers and command-line users, and the API for them may change in future versions.  Please see the bottom of this post for information about how to contribute to development if you'd like to help improve multiwallet mode and make it available in the graphical interface.  For more information about multiwallet mode, please see the [release notes][].
+
+## Performance improvements
+
+As part of the continuing effort to make full nodes available to as many users as possible even as the block chain continues to grow in size and complexity, Bitcoin Core 0.15.0 includes several significant performance improvements.
+
+- **30% to 40% faster block validation and 10% to 20% less memory used** on tests of Initial Block Download (IBD), with far fewer writes to disk.  This is the result of simplifying the format of the the chainstate database that tracks each spendable group of bitcoins and what information the owner of those bitcoins needs to provide in order to spend them.
+
+- **40% to 50% faster validation of blocks consisting of previously-seen transactions** as the result of repeating fewer validation steps when a previously-verified mempool transaction is later received in a block.
+
+- **Moderate performance gains on some platforms** as the result of using hardware acceleration for some operations, such as support on modern computer processors for single-SHA256 hashing which results in about a 50% improvement each time one of Bitcoin's double-SHA256 hashes needs to be computed.  This mainly benefits users of 64-bit Intel and AMD processors produced in 2008 or later.
```

Comment:
Hmm, this is misleading due to the fact that this is disabled-by-default (and in the release binaries). Maybe mention it is just "in testing" or so?

Reply:
I don't feel comfortable saying something is in testing in a post about a *release*, so in the edit commit I'm about to push, I changed the example to mention the LevelDB CRC acceleration instead.  It's less of a cool example IMO, but thanks for not letting me get away with misleading readers.